### PR TITLE
Add activity as per #752

### PR DESCRIPTION
**Pull request by arvidj**

### CODE REVIEW

File: examples/hello/print/print_display/testcase_list/input.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ straightforward:
 
 {testcase_list.play}
 
+### Activity
+
+Try changing to program so that the index of each element in the vector is also printed. The new output should look like this:
```
Comment (mdinger) [inline examples/hello/print/print_display/testcase_list/input.md]: `s/changing to/changing the/`

File: examples/hello/print/print_display/testcase_list/input.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ straightforward:
...
+
+```
+[0: 10, 1: 40, 2: 30].
```
Comment (mdinger) [inline examples/hello/print/print_display/testcase_list/input.md]: This shouldn't have a period

File: examples/hello/print/print_display/testcase_list/input.md [hunk 3]
```diff
@@ -16,6 +16,15 @@ straightforward:
 
 {testcase_list.play}
 
+### Activity
+
+Try changing to program so that the index of each element in the vector is also printed. The new output should look like this:
+
+```
```
Comment (mdinger) [inline examples/hello/print/print_display/testcase_list/input.md]: Should be triple backticks followed by rust (can't escape backticks in markdown so I can't just write it out). See line 8.

**Comment by mdinger** [comment]
Thanks!

**Comment by steveklabnik** [comment]
Same, thank you!

**Pull request merged**